pkg/virt-handler/vm.go

```diff
@@ -291,16 +291,32 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 				domainInterfaceStatusByMac[domainInterfaceStatus.Mac] = domainInterfaceStatus
 			}
 
+			existingInterfacesSpecByName := map[string]v1.Interface{}
```

Comment:
We named a type "Interface"? 😱

Reply:
`NetworkInterface` might have been a better name, but at least that matches the name in the domain xml. :D